### ISSUE TITLE
Is the aria-label usefull on navigation

### ISSUE THREAD

**Issue by alienlebarge** [vdch/regenair-fabricator#389]
## Description
The nav molecule has a `aria-label`. Is it usefull ?
Aria should be used to solve HTML lacks.

```html
<nav class="nav nav-inline hidden-sm-down" aria-label="sous-navigation">
  ...
</nav>
```

## Ask
Would it be better to use something like that ?

```html
<nav class="nav nav-inline hidden-sm-down">
    <h2 class="sr-only">Sous-navigation</h2>
    ...
</nav>
```

If the CSS breaks we get something more usable
![napkin 3 17 11 16 a 1 02 12 pm](https://cloud.githubusercontent.com/assets/611234/20388621/01ab5458-acc6-11e6-8b84-ff70a3d4e3e2.png)
